bin/nuocd

```diff
@@ -9,8 +9,16 @@ die () { echo "$*"; exit 1; }
 
 _python=${PYTHON}
 if test -z "$_python"; then
-  _python=$(command -v python2.7) || _python=$(command -v python2) || _python=$(command -v python)  \
-  || die "Python must be installed"
+    _python=$(command -v python3) \
+	|| $(command -v python2.7) \
```

Comment:
Shouldn't this be `|| _python="$(command -v python2.7)"`?

Reply:
probably, I've been testing by setting the variable PYTHON as I don't have python2 with without python3 installed.  I will confirm.